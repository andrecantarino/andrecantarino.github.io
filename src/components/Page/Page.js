import React from 'react'
import styled from 'styled-components'
import { Header } from '../Header/Header'
import { Footer } from '../Footer/Footer'

const StyledPage = styled.div`
  display: flex;
  flex-flow: row wrap;

  main {
    display: flex;
    flex-flow: row wrap;
    height: 86vh;
    width: 100%;
    background-color: var(--backgroundPrimary);
  }
`

const Page = ({ children }) => {
  return (
    <StyledPage>
      <Header />
      <main>
        { children }
      </main>
      <Footer />
    </StyledPage>
  )
}

export default Page