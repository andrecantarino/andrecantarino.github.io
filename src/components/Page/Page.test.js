import React from 'react'
import { render } from '@testing-library/react'
import Page from './Page'

describe('Page | snapshot', () => {
  it('should take a snapshot', () => {
    const { asFragment } = render(<Page />)

    expect(asFragment(<Page />)).toMatchSnapshot()
  })
});