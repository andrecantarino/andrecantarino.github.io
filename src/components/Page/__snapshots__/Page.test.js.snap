// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page | receive content should receive content as a children 1`] = `
<DocumentFragment>
  <div
    class="sc-dkzDqf ipNBYu"
  >
    <header
      class="sc-bczRLJ fPvdEA"
    />
    <main>
      <div>
        <p>
          Hello World
        </p>
      </div>
    </main>
    <footer
      class="sc-gsnTZi ZOYfm"
    />
  </div>
</DocumentFragment>
`;

exports[`Page | snapshot should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-dkzDqf ipNBYu"
  >
    <header
      class="sc-bczRLJ fPvdEA"
    />
    <main />
    <footer
      class="sc-gsnTZi ZOYfm"
    />
  </div>
</DocumentFragment>
`;
